apps/block_scout_web/lib/block_scout_web/templates/transaction/overview.html.eex

```diff
@@ -62,179 +41,361 @@
+                  </dt>
+                  <dd class="col-sm-9 col-lg-10 transfers-list-mobile-container">
+                  <%= for minting <- mintings do %>
+                    <%= render BlockScoutWeb.TransactionView, "_total_transfers_from_to.html", Map.put(assigns, :transfer, minting) %>
+                  <% end %>
+                  </dd>
+                </dl>
+              <% end %>
+              <!-- Tokens Burnt -->
+              <%= if Enum.count(burnings) > 0 do %>
+                <dl class="row">
+                  <dt class="col-sm-3 col-lg-2 text-muted">
+                    <%= render BlockScoutWeb.CommonComponentsView, "_i_tooltip_2.html",
+                      text: gettext("List of token burnt in the transaction.") %>
+                    <%= gettext "Tokens Burnt" %></dt>
+                  <dd class="col-sm-9 col-lg-10 transfers-list-mobile-container">
+                  <%= for burning  <- burnings do %>
+                    <%= render BlockScoutWeb.TransactionView, "_total_transfers_from_to.html", Map.put(assigns, :transfer, burning) %>
+                  <% end %>
+                  </dd>
+                </dl>
+              <% end %>
+
+              <%= if Enum.count(creations) > 0 do %>
+                <dl class="row">
+                  <dt class="col-sm-3 col-lg-2 text-muted">
+                    <%= render BlockScoutWeb.CommonComponentsView, "_i_tooltip_2.html",
+                      text: gettext("List of ERC-1155 tokens created in the transaction.") %>
+                    <%= gettext "Tokens Created" %></dt>
+                  <dd class="col-sm-9 col-lg-10 transfers-list-mobile-container">
+                  <%= for creation  <- creations do %>
+                    <%= render BlockScoutWeb.TransactionView, "_total_transfers_from_to.html", Map.put(assigns, :transfer, creation) %>
+                  <% end %>
+                  </dd>
+                </dl>
+              <% end %>
+            <% _ -> %>
           <% end %>
-          <!-- feeRecipient -->
+          <!-- Value -->
           <dl class="row">
-            <dt class="col-sm-3 text-muted"><%= gettext "Fee Recipient" %> </dt>
-            <dd class="col-sm-9" data-selector="fee-recipient">
-              <%= if @transaction.gas_fee_recipient_hash do %>
-                <%= link(
-                      @transaction.gas_fee_recipient_hash,
-                      class: "address__link",
-                      to: address_path(@conn, :show, @transaction.gas_fee_recipient_hash)
-                    ) %>
-              <% else %>
-                None
-              <% end %>
+            <dt class="col-sm-3 col-lg-2 text-muted">
+              <%= render BlockScoutWeb.CommonComponentsView, "_i_tooltip_2.html",
+                text: gettext("The value being transacted in Ether and fiat value. Note: You can click the fiat value (if available) to see historical value at the time of transaction.") %>
```

Comment:
Should we change `Ether` to `CELO` here? I understand that it's used as unit name but still looks confusing.